src/main/java/by/yurachel/web_app/repository/PhoneRepository.java

```diff
@@ -3,24 +3,26 @@
 import by.yurachel.web_app.entity.Phone;
 
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 
 public class PhoneRepository {
     private static final PhoneRepository INSTANCE = new PhoneRepository();
-
     private final List<Phone> phones = new ArrayList<>();
+    public static final List<Integer> ID = new ArrayList<>(Arrays.asList(1, 2, 3, 4, 5));
```

Comment:
You are editing this field, so it should be final.
Also it should be private with getters/setters (or other access methods)